src/unix/pthread.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) the libgit2 contributors. All rights reserved.
+ *
+ * This file is part of libgit2, distributed under the GNU GPL v2 with
+ * a Linking Exception. For full terms see the included COPYING file.
+ */
+
+#ifndef GIT_PTHREAD_H
+#define GIT_PTHREAD_H
```

Comment:
minor nit: our header guards are of the form `INCLUDE_unix_pthread_h__`.

I don't know the history here but we are reasonably consistent 😀